src/components/Cover/Cover.module.css

```diff
@@ -0,0 +1,68 @@
+.wrapper {
+    background-image: url("../../images/coverOverlay.svg"), url("../../images/coverPhoto.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+    min-height: 200px;
+    width: 100%;
+}
+
+.title {
+    max-width: 238px;
+    margin: auto;
+    padding-top: 33px;
```

Comment:
This should be a `margin` with `overflow: auto` on a parent.
Without `overflow` this will not work because of [collapsing margins](https://stackoverflow.com/questions/13573653/css-margin-terror-margin-adds-space-outside-parent-element)

Reply:
https://stackoverflow.com/questions/13573653/css-margin-terror-margin-adds-space-outside-parent-element

Reply:
fixed